styles/common.module.scss

```diff
@@ -0,0 +1,145 @@
+@timesnewroman: "Times New Roman", Times, Serif;
```

Comment:
We can keep this file but please pare it down to just the bare minimum.

Reply:
done